pnpm-lock.yaml

```diff
@@ -15157,31 +15297,42 @@ snapshots:
       '@types/unist': 3.0.3
       vfile-message: 4.0.2
 
-  vite-plugin-wasm@3.4.1(vite@6.2.3(@types/node@20.17.22)(terser@5.31.1)(yaml@2.7.0)):
+  vite-plugin-wasm@3.4.1(vite@6.2.4(@types/node@22.13.15)(terser@5.31.1)(yaml@2.7.1)):
     dependencies:
-      vite: 6.2.3(@types/node@20.17.22)(terser@5.31.1)(yaml@2.7.0)
+      vite: 6.2.4(@types/node@22.13.15)(terser@5.31.1)(yaml@2.7.1)
 
-  vite-tsconfig-paths@5.1.4(typescript@5.8.2)(vite@6.2.3(@types/node@20.17.22)(terser@5.31.1)(yaml@2.7.0)):
+  vite-tsconfig-paths@5.1.4(typescript@5.8.2)(vite@6.2.4(@types/node@22.13.15)(terser@5.31.1)(yaml@2.7.1)):
     dependencies:
       debug: 4.4.0
       globrex: 0.1.2
       tsconfck: 3.1.0(typescript@5.8.2)
     optionalDependencies:
-      vite: 6.2.3(@types/node@20.17.22)(terser@5.31.1)(yaml@2.7.0)
+      vite: 6.2.4(@types/node@22.13.15)(terser@5.31.1)(yaml@2.7.1)
     transitivePeerDependencies:
       - supports-color
       - typescript
 
-  vite@6.2.3(@types/node@20.17.22)(terser@5.31.1)(yaml@2.7.0):
+  vite@6.2.4(@types/node@20.17.29)(terser@5.31.1)(yaml@2.7.1):
+    dependencies:
+      esbuild: 0.25.2
+      postcss: 8.5.3
+      rollup: 4.38.0
+    optionalDependencies:
+      '@types/node': 20.17.29
+      fsevents: 2.3.3
+      terser: 5.31.1
+      yaml: 2.7.1
+
+  vite@6.2.4(@types/node@22.13.15)(terser@5.31.1)(yaml@2.7.1):
```

Comment:
CI is failing because `@types/node` gets resolved to v22 again for some reason, I'll take a look at this later.

We have an Electron update in the pipeline which will bump Node.js to v22 anyway.

https://github.com/gravitational/teleport/actions/runs/14194542054/job/39766527263?pr=53613#step:7:9

Reply:
What I did was similar to the previous instance of this problem, but this time I didn't have to commit the override. https://github.com/gravitational/teleport/pull/51767#discussion_r1953991637

I added the override to package.json (`"vite>@types/node": "^20.17.29",`), then ran `pnpm i`, then removed the override and ran `pnpm i` again. This resulted in pnpm-lock.yaml where vite uses @types/node v20 and not v22.